kademlia/routing.py

```diff
@@ -1,8 +1,9 @@
 import heapq
 import time
 import operator
-from collections import OrderedDict
+import asyncio
 
+from collections import OrderedDict
```

Comment:
Ooops. Nothing serious. That's a little trick that I don't know why git can't discover my change so I have to make another "change" to remind it. If you think it's not good, I would try to fix it. : )